WatchFaceAlphaKotlin/app/build.gradle

```diff
@@ -54,34 +54,25 @@ android {
 dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.3.2'
-
-    // Dependencies for working with Architecture components
-
-    // Room components
-    implementation "androidx.room:room-ktx:$rootProject.roomVersion"
-    kapt "androidx.room:room-compiler:$rootProject.roomVersion"
-    androidTestImplementation "androidx.room:room-testing:$rootProject.roomVersion"
-
-    // Lifecycle components
-    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$rootProject.lifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-livedata-ktx:$rootProject.lifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-common-java8:$rootProject.lifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-extensions:$rootProject.lifecycleVersion"
+    implementation 'androidx.core:core-ktx:1.6.0'
 
     // Kotlin components
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     api "org.jetbrains.kotlinx:kotlinx-coroutines-core:$rootProject.coroutines"
     api "org.jetbrains.kotlinx:kotlinx-coroutines-android:$rootProject.coroutines"
 
-    implementation "androidx.wear:wear-watchface:$rootProject.watchFaceAlphaVersion"
 
-    implementation "androidx.wear:wear-watchface-client:$rootProject.watchFaceAlphaVersion"
-    implementation "androidx.wear:wear-watchface-complications-rendering:$rootProject.watchFaceAlphaVersion"
+    // Wear libraries
+    implementation 'androidx.wear:wear:1.1.0'
 
+    // Watch face specific libraries
     implementation "androidx.wear:wear-complications-data:$rootProject.watchFaceAlphaVersion"
     implementation "androidx.wear:wear-complications-provider:$rootProject.watchFaceAlphaVersion"
```

Comment:
Don't want to delay this PR, but the latest version of `complications` changed this dependency to `wear-complications-data-source`, which will require some API updates as a follow-up.

Reply:
Yes, I will update that in a follow up PR to get this one live. Should be easier to review then